src/search/search_engine.h

```diff
@@ -31,6 +31,7 @@ class SuccessorGenerator;
 enum SearchStatus {IN_PROGRESS, TIMEOUT, FAILED, SOLVED};
 
 class SearchEngine {
+    std::string description;
```

Comment:
While I like that search engines get a description, it is not used at the moment right? But I guess it will be after isseu1040 is merged?

Reply:
I checked this again and it is used in the new code of iterated search to print the message `Starting search: ...` Currently, the strings are passed directly to iterated search, so this is not necessary.